packages/rocketchat-livechat/app/client/lib/_visitor.js

```diff
@@ -58,7 +58,10 @@ this.visitor = new class {
 
 				// notification sound
 				if (Session.equals('sound', true) && msg.u._id !== Meteor.userId()) {
-					$('#chatAudioNotification')[0].play();
+					const audioVolume = Meteor.user().settings.preferences.notificationsSoundVolume || 100;
+					const audio = $('#chatAudioNotification')[0];
```

Comment:
you can use `document.getElementById` here, it's faster if you won't use jQuery.